integration-libs/cdc/root/guards/cdc-auth-logout.service.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { CdcLogoutGuard } from './cdc-logout.guard';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CdcAuthLogOutService extends CdcLogoutGuard {
```

Comment:
a service should not be a child class of a guard, let's follow **composition over Inheritance** principle

Reply:
Adapted code to let this service not extends a guard. Please have a check.